[2024-04-26T22:53:25.456+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-26T22:53:25.483+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Crypto_ETL.notify manual__2024-04-26T22:53:12.095230+00:00 [queued]>
[2024-04-26T22:53:25.492+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Crypto_ETL.notify manual__2024-04-26T22:53:12.095230+00:00 [queued]>
[2024-04-26T22:53:25.494+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-26T22:53:25.507+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): notify> on 2024-04-26 22:53:12.095230+00:00
[2024-04-26T22:53:25.515+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=660) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-26T22:53:25.517+0000] {standard_task_runner.py:63} INFO - Started process 662 to run task
[2024-04-26T22:53:25.518+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Crypto_ETL', 'notify', 'manual__2024-04-26T22:53:12.095230+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/processing_ETL.py', '--cfg-path', '/tmp/tmproay9ex0']
[2024-04-26T22:53:25.519+0000] {standard_task_runner.py:91} INFO - Job 138: Subtask notify
[2024-04-26T22:53:25.569+0000] {task_command.py:426} INFO - Running <TaskInstance: Crypto_ETL.notify manual__2024-04-26T22:53:12.095230+00:00 [running]> on host 7542c54372fc
[2024-04-26T22:53:25.664+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Crypto_ETL' AIRFLOW_CTX_TASK_ID='notify' AIRFLOW_CTX_EXECUTION_DATE='2024-04-26T22:53:12.095230+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-26T22:53:12.095230+00:00'
[2024-04-26T22:53:25.666+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-26T22:53:25.667+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-26T22:53:25.668+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Ceci est la fin de l entrainement !"']
[2024-04-26T22:53:25.680+0000] {subprocess.py:86} INFO - Output:
[2024-04-26T22:53:25.682+0000] {subprocess.py:93} INFO - Ceci est la fin de l entrainement !
[2024-04-26T22:53:25.682+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-26T22:53:25.684+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-26T22:53:25.710+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Crypto_ETL, task_id=notify, execution_date=20240426T225312, start_date=20240426T225325, end_date=20240426T225325
[2024-04-26T22:53:25.733+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-26T22:53:25.751+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-26T22:53:25.753+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
